low-level/message_handlers/node_data_msg_handler.py

```diff
@@ -529,84 +572,99 @@ def _generate_cpu_data(self):
             # Assigning default value to _cpu_usage_threshold
             self._cpu_usage_threshold = self.DEFAULT_CPU_USAGE_THRESHOLD
 
-        if self._node_sensor.cpu_usage >= self._cpu_usage_threshold \
-           and not self.high_usage['cpu'] :
-
-            self.high_usage['cpu'] = True
-            # Create the cpu usage data message and hand it over to the egress processor to transmit
-
-            fault_event = "CPU usage increased to %s, beyond configured threshold of %s" \
-                            %(self._node_sensor.cpu_usage, self._cpu_usage_threshold)
-            logger.warning(fault_event)
-
-            # Create the local mount data message and hand it over to the egress processor to transmit
-            cpuDataMsg = CPUdataMsg(self._node_sensor.host_id,
-                                self._epoch_time,
-                                self._node_sensor.csps,
-                                self._node_sensor.idle_time,
-                                self._node_sensor.interrupt_time,
-                                self._node_sensor.iowait_time,
-                                self._node_sensor.nice_time,
-                                self._node_sensor.softirq_time,
-                                self._node_sensor.steal_time,
-                                self._node_sensor.system_time,
-                                self._node_sensor.user_time,
-                                self._node_sensor.cpu_core_data,
-                                self._node_sensor.cpu_usage,
-                                self.FAULT,
-                                fault_event
-                            )
-
-            # Add in uuid if it was present in the json request
-            if self._uuid is not None:
-                cpuDataMsg.set_uuid(self._uuid)
-            jsonMsg = cpuDataMsg.getJson()
-            self.cpu_sensor_data = jsonMsg
-            self.os_sensor_type["cpu_usage"] = self.cpu_sensor_data
+        cpu_persistent_data = self.read_persistent_data('CPU_USAGE_DATA')
+        if cpu_persistent_data.get('cpu_usage_time_map') is not None:
+            previous_check_time = int(cpu_persistent_data['cpu_usage_time_map'])
+        else:
+            previous_check_time = int(-1)
+        self.usage_time_map['cpu'] = current_time

```

Comment:
you can directly use time.time() here. One liner functions are not generally useful (`get_current_time`)

Reply:
It is used in multiple places, for example service_monitor.py, so @cdeshmukh asked to define it in Utility class and it should be used from there if needed elsewhere.